src/lexer/Lexer.ts

```diff
@@ -457,7 +455,7 @@ export class Lexer {
 
         //replace escaped quotemarks "" with a single quote
         value = value.replace(/""/g, '"');
-        this.addToken(TokenKind.StringLiteral, new BrsString(value));
```

Comment:
The concept of a `literal` type on tokens has been removed, because we don't need it anymore.